include/clap/ext/audio-ports-config.h

```diff
@@ -72,6 +72,13 @@ typedef struct clap_plugin_audio_ports_config {
 
 typedef struct clap_plugin_audio_ports_config_info {
   // get info about about an audio port.
+
+  // gets the clap_id of the currently selected config
+  // clap_id might now match the index, can be used to call
+  // clap_plugin_audio_ports_config::select()
+  // [main-thread]
+  clap_id(CLAP_ABI* current_config)(const clap_plugin_t* plugin);
```

Comment:
Is this needed? The host is the one selecting a config. So it knows the current config.

Reply:
not necessarily - the plugin itself could also switch configurations (like in a user selection, as the commentary states) and the host would not know which one is selected now.